dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jaxws.common-2.3.feature

```diff
@@ -1,46 +1,66 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=com.ibm.websphere.appserver.internal.jaxws-2.3
+symbolicName=com.ibm.websphere.appserver.jaxws.common-2.3
+WLP-DisableAllFeatures-OnConflict: false
```

Comment:
Should this be false?  This is a new feature that hasn't shipped yet.  Or are you doing this because you are expecting this to become jaxws-2.2 eventually and you want to be prepared for it being made 2.2?

Reply:
Exactly, I'm matching existing behavior where I can. If I need to reverse it I can.

Reply:
I've removed it.